catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/HitCountRequestImpl.java

```diff
@@ -41,23 +41,23 @@ public State complete(org.omg.CORBA.IntHolder number_of_hits)
   }
 
   @Override
-  public RequestDescription get_request_description() throws ProcessingFault, SystemFault {
```

Comment:
❗️ You can't rename these methods. They override interface methods in generated CORBA api code.